Add-ons/Starter-Kit-TEMP/Widgets/index.md

```diff
@@ -208,3 +208,31 @@ The form has the following settings available:
 Below you can see how a ***Contact Form*** will look per default:
 
 ![movePage.jpg](images/Contact-form-example.png)
+
+### The Logos Widget
+
+The Logos Widget does what the name implies and lets you set up logos, the settings for your logos images.
```

Comment:
Not sure about the last bit of this article - I've made the following suggestion instead:

```suggestion
The Logos Widget does what the name implies and lets you add a set of logos to your website.
```